nemo/collections/llm/README.md

```diff
@@ -0,0 +1,13 @@
+NeMo LLM Collection
+===================
+
+The NeMo LLM Collection is the first collection that features NeMo 2.0, a redesign of NeMo that
+aims to improve the user experience in by taking a more PyTorch Lightning-like approach to make
+NeMo both simpler and more modular. 
+
+The current models are currently reimplemented in 2.0 as part of this collection:
```

Comment:
The following models are currently reimplemented in 2.0 as part of this collection: